docs/applications/simple-bank-account/README.md

```diff
@@ -122,7 +122,7 @@ scalar.dl.client.private_key_path=conf/client-key.pem
 If everything is set up properly you should be able to register your certificate on the Scalar DL network as
 
 ```bash
-$ ${SCALAR_SDK_HOME}/client/bin/register-cert -properties ./conf/client.properties
+$ ${SCALAR_SDK_HOME}/client/bin/register-cert --properties=./conf/client.properties
```

Comment:
= and space both work. so let's use space for now.

Reply:
Thank you for review comments.
Changes updated in eab64cd6b0cdbede35a100618a5e8a7ad30fcd81